code/modules/paperwork/photocopier.dm

```diff
@@ -120,7 +120,7 @@
 		else if(doccopy)
 			for(var/i = 0, i < copies, i++)
 				if(toner > 5 && !busy && doccopy)
-					new /obj/item/documents/photocopy(src, doccopy)
+					new /obj/item/documents/photocopy(loc)

```

Comment:
not really, the cases are:

- Steal any set of secret documents: there's only one photocopy item that can be produced by it
- Steal a specific set of secret documents: no photocopy would be accepted anyway.